### DISCUSSION TITLE
Welcome to Face-Recognition-Attendance-System Discussions!

### DISCUSSION THREAD

**Discussion by kmhmubin** [kmhmubin/Face-Recognition-Attendance-System#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Parth31-10** [comment]
when i tried to install python-csv, then it is showing microsoft visual studio c++ is required. please help me to get it solve.

**Reply by kmhmubin** [reply]
If you use the latest python version, then you don't need to install **CSV**. Just import CSV. It will solve the problem.

**Reply by Parth31-10** [reply]
i am using python 3.7.0 version.

**Comment by Parth31-10** [comment]
[ WARN:0] global C:\Users\appveyor\AppData\Local\Temp\1\pip-req-build-6lylwdcz\opencv\modules\videoio\src\cap_msmf.cpp (435) `anonymous-namespace'::SourceReaderCB::~SourceReaderCB terminating async callback

**Reply by Parth31-10** [reply]
i am getting this error please help me out

**Reply by kmhmubin** [reply]
> [ WARN:0] global C:\Users\appveyor\AppData\Local\Temp\1\pip-req-build-6lylwdcz\opencv\modules\videoio\src\cap_msmf.cpp (435) `anonymous-namespace'::SourceReaderCB::~SourceReaderCB terminating async callback

https://stackoverflow.com/questions/59596748/warn0-global-sourcereadercbsourcereadercb-terminating-async-callback-wa

**Reply by Parth31-10** [reply]
again the same problem is occuring even making the necessary changes.
[ WARN:0] global C:\Users\appveyor\AppData\Local\Temp\1\pip-req-build-6lylwdcz\opencv\modules\videoio\src\cap_msmf.cpp (435) `anonymous-namespace'::SourceReaderCB::~SourceReaderCB terminating async callback

**Comment by sachinshukla7944** [comment]
C:\Users\sachi\AppData\Local\Programs\Python\Python39\python.exe C:/Users/sachi/PycharmProjects/Face-Recognition-Attendance-System/FRAS/main.py
	**********************************************
	***** Face Recognition Attendance System *****
	**********************************************

********** WELCOME MENU **********
[1] Check Camera
[2] Capture Faces
[3] Train Images
[4] Recognize & Attendance
[5] Auto Mail
[6] Quit
Enter Choice: 1
**Traceback (most recent call last):
  File "C:\Users\sachi\PycharmProjects\Face-Recognition-Attendance-System\FRAS\main.py", line 101, in <module>
    mainMenu()
  File "C:\Users\sachi\PycharmProjects\Face-Recognition-Attendance-System\FRAS\main.py", line 38, in mainMenu
    checkCamera()
  File "C:\Users\sachi\PycharmProjects\Face-Recognition-Attendance-System\FRAS\main.py", line 68, in checkCamera
    check_camera.camera()
  File "C:\Users\sachi\PycharmProjects\Face-Recognition-Attendance-System\FRAS\check_camera.py", line 16, in camera
    gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)






cv2.error: OpenCV(4.5.3) C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-1i5nllza\opencv\modules\imgproc\src\color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cv::cvtColor'


Process finished with exit code 1**



> I'm trying to fix this problem but can't , please help !

**Comment by oavjnp** [comment]
Want to modify the code for android phones with termux. Any suggestions please.
